components/mjs/a11y/util-pack.js

```diff
@@ -1,12 +1,12 @@
-import {Loader} from '../../../js/components/loader.js';
+import {Loader} from '#js/components/loader.js';
 import '../input/mml/init.js';
 import {Sre} from './sre/sre.js';
 import './semantic-enrich/semantic-enrich.js';
 import './explorer/explorer.js';
-import MathMaps from '../../../js/a11y/mathmaps.js';
-import base from 'speech-rule-engine/lib/mathmaps/base.json';
-import en from 'speech-rule-engine/lib/mathmaps/en.json';
-import nemeth from 'speech-rule-engine/lib/mathmaps/nemeth.json';
+import {MathMaps} from '#js/a11y/mathmaps.js';
+import base from 'speech-rule-engine/lib/mathmaps/base.json' assert { type: 'json' };
+import en from 'speech-rule-engine/lib/mathmaps/en.json' assert {type: 'json' };
+import nemeth from 'speech-rule-engine/lib/mathmaps/nemeth.json' assert {type: 'json' };
```

Comment:
I see, that is here now.

Reply:
This is the file used by webpack.

I suppose I could have isolated just these four lines plus the setting of the maps into a second utility file and used three versions of that instead.  I can do that if you prefer.